### ISSUE TITLE
tacotron2 result may have noise like any machine sound 

### ISSUE THREAD

**Issue by Ellie1013** [TensorSpeech/TensorFlowTTS#636]
I trained korean tacotron2 at 468K using kss data and my custom data.
And I made a wave result with mb-melgan.
The result is great, but it has noise like any machine sound.

One of machine sound that the result have is below...
https://drive.google.com/file/d/17FDKD2Yp_cWLsWsfqdKk-sWc1uuk_Gm3/view?usp=sharing

My custom data doesn't have this machine sound, it is very clear.
I trained tacotron2 with kss at 60K, and did transfer learning with the custom data at 468K.
What problem? Please help me..

The environment:
tensorflow 2.4
python 3.8
CUDA 11.1
cudnn 8

**Comment by samuel-lunii** [comment]
@Ellie1013 Did you train MB-MelGAN with your own dataset ?

**Comment by Ellie1013** [comment]
@samuel-lunii no, I use pretrained mb-melgan from https://huggingface.co/tensorspeech/tts-mb_melgan-kss-ko

**Comment by dathudeptrai** [comment]
@Ellie1013 please provide a full audio in real inference on testset. :D.

**Comment by Ellie1013** [comment]
@dathudeptrai  there are two samples, the last part has noise with normal sound in first one, 
and the middle part has noise alone in second one:)
https://drive.google.com/file/d/1xgMMslT7NDWaTtBF_o5HQIYeXAvIleZ7/view?usp=sharing
https://drive.google.com/file/d/1tmre07WEZeCsme9jbItdEUznM4Rf9-YU/view?usp=sharing

**Comment by dzcmingdi** [comment]
> @dathudeptrai there are two samples, the last part has noise with normal sound in first one, and the middle part has noise alone in second one:) https://drive.google.com/file/d/1xgMMslT7NDWaTtBF_o5HQIYeXAvIleZ7/view?usp=sharing https://drive.google.com/file/d/1tmre07WEZeCsme9jbItdEUznM4Rf9-YU/view?usp=sharing

Do you solve it?
I'm confused with this problem now.
